tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -14634,6 +14634,7 @@ TEST_F(EnergyPlusFixture, Test_UnitarySystemModel_SubcoolReheatCoil)
     state->dataScheduleMgr->Schedule(6).CurrentValue = 1.0; // Enable schedule without calling schedule manager
     state->dataScheduleMgr->Schedule(7).CurrentValue = 4.0; // Enable schedule without calling schedule manager
 
+    state->dataEnvrn->OutBaroPress = 101325;
```

Comment:
Yes, I have seen many unit tests forget to initialize OA pressure. If you don't single step into the code you will likely miss this initialization. The adjustments below look reasonable.